rs/ledger_suite/icp/src/lib.rs

```diff
@@ -1134,57 +1132,28 @@ pub struct IterBlocksRes(pub Vec<EncodedBlock>);
 pub struct BlockArg(pub BlockIndex);
 pub struct BlockRes(pub Option<Result<EncodedBlock, CanisterId>>);
 
-fn get_block_indices(
-    blocks_len: usize,
+// A helper function for archive_node/get_blocks endpoints
```

Comment:
Contrary to what the comment says, this is now used by both archives and ledgers, right?

Reply:
I suppose you refer to this [remark](https://github.com/dfinity/ic/pull/3836#discussion_r1975625844): if so, let's continue the discussion on that thread.